src/x-platform/electron.js

```diff
@@ -17,6 +18,24 @@ if (window.require) {
 
 const notificationClickTimeout = 20000
 
+// http://crocodillon.com/blog/parsing-emoji-unicode-in-javascript
+const emojiRegExp = new RegExp([
```

Comment:
huh! and this is good point!
we don't need it because we're getting an emoji from the server, Probably I can save raw text and use it, this will be much more stable.
Thanks, please don't merge yet.